### PR TITLE
login history design

### PR DESCRIPTION
**Pull request by CreedTech**
completed login history design

### CODE REVIEW

**Comment by CreedTech** [comment]
@deviator53  @ELDEVODE  @Olufade-James

**Comment by deviator53** [comment]
- Resolve the conflict @CreedTech

**Comment by CreedTech** [comment]
@deviator53  conflicts resolved
Help me verify

File: src/App.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { Routes, Route } from "react-router-dom";
 import { ThemeProvider } from "@mui/material";
 
-import { Home, Login, LoginHistory, UserProfile } from "./pages";
+import { Home, Login, Signup, UserProfile, ForgotPassword, Otp } from "./pages";
```
Comment (deviator53) [inline src/App.tsx]: @CreedTech you didn't include LoginHistory ?

Comment (CreedTech) [reply src/App.tsx]: i have added it just now @deviator53  didn't notice i removed it again

**Review by deviator53 — APPROVED** [review]

**Review by Yagazie-davidson — APPROVED** [review]

**Pull request merged**